templates/generic/action.html

```
@@ -1,6 +1,6 @@
 {% if action_url %}
-  <a {% if not disabled %}href="{{ action_url }}"{% endif %} class="btn btn-sm join-item {{ button_style|default_if_none:"btn-ghost" }}{% if disabled %} pointer-events-none{% endif %}" {% if disabled %}aria-disabled="true"{% endif %} {% if title %}title="{{ title }}"{% endif %} {% if not disabled and open_url_in_new_tab %}target="_blank" rel="noopener noreferrer"{% endif %}>
-    {{ label }}
+  <a {% if not disabled %}href="{{ action_url }}"{% endif %} class="btn btn-sm join-item {{ button_style|default_if_none:"btn-ghost" }}{% if truncate %} max-w-xs{% endif %}{% if disabled %} pointer-events-none{% endif %}" {% if disabled %}aria-disabled="true"{% endif %} {% if title %}title="{{ title }}"{% endif %} {% if not disabled and open_url_in_new_tab %}target="_blank" rel="noopener noreferrer"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)